contents/handbook/people/team-structure/platform.md

```diff
@@ -9,28 +9,62 @@ hideAnchor: true
 
 [See team structure page](/handbook/people/team-structure/team-structure)
 
-## Mission
+# Platform Sub-Team Mission Statements
 
+## Ingestion
 
-Make deploying, scaling and managing PostHog easy
+**Provide the best events pipeline in the world.**
 
-## Goals & Key Results
+## Infrastructure
 
-* PostHog is the easiest self-hosted product to deploy and scale in the world
-    * Result: Volume of clients requiring help @marcushyett-ph
-    * Result: % of successful installs / upgrades @guidoiaquinti
-    * Result: Max size of operating instances @fuziontech
-    * Monitor: % of users on latest version of PostHog @tiina303
-    * Result: We don't lose customers because we can't scale enough (shared with Core Analytics) @marcushyett-ph
-* Move all posthog-supported instances to self-hosted
-    * Result: No alerts come to us from customer-instances
-* All valid events are ingested correctly
-    * Result: Time ingestion is down on cloud (from alert) @fuziontech
-    * Result: Lost events between ingress and clickhouse @fuziontech
-* Data is as up to date as possible
-    * Result: Latency between ingress and clickhouse @fuziontech
-* A good experience for plugin developers
-    * Result: Good docs, alerting, debugging and testing @yakkomajuri
+**Make deploying, scaling, and managing PostHog easy, fast, and reliable.**
+
+# Roadmap
```

Comment:
Dropped from this focus: client libraries.

Reply:
You are correct! Nice 👀